src/backend/executor/execTuples.c

```diff
@@ -359,6 +361,12 @@ ExecSetSlotDescriptor(TupleTableSlot *slot,		/* slot to change */
 
 		MemoryContextSwitchTo(oldcontext);
 	}
+	slot->slot_deform_tuple_gen_info.code_generator = SlotDeformTupleCodeGen_Enroll(slot,
```

Comment:
Enrollment should also be in the ifdef. Or if we decide to put ifdef in wrapper, we need to do so uniformly.